src/ElectronNET.WebApp/Views/About/Index.cshtml

```diff
@@ -20,7 +20,7 @@
 {
     return WebHost.CreateDefaultBuilder(args)
         .UseElectron(args)
-        .UseStartup<Startup>()
+        .UseStartup&lt;Startup>()
```

Comment:
Would prefer the `>` to be also transformed (`&gt;`) - for consistency / clarity. Same for the other occurrence.